system/core/tags.php

```diff
@@ -0,0 +1,192 @@
+<?php
+	// TODO:
+	// - add styling for default theme
```

Comment:
I believe this TODO is resolved with Tammo's commits? The rest below will probably stay until after this first version is merged.